aten/src/ATen/native/cudnn/Conv.cpp

```diff
@@ -390,7 +390,7 @@ std::vector<perf_t> getValidAlgorithms(perf_t *perfResults, const ConvolutionArg
 #if CUDNN_VERSION < 7500
   bool blacklist = std::is_same<decltype(perfResults[0].algo), cudnnConvolutionBwdDataAlgo_t>::value;
   int stride_dim = args.input.dim() - 2;
-  blacklist &&= std::any_of(std::begin(args.params.stride),
```

Comment:
Just curious, why did this compile?

Reply:
Oh, do we not have CUDNN < 7.5 in our CI?

Reply:
You are right, we don't have CUDNN < 7.5 in our CI